README.md

```diff
@@ -0,0 +1,26 @@
+# **Supply Chain**
+
+## Project Description
+Feeding San Diego helps reduce food waste by purchasing food from suppliers and distributors to give to those in need. This project organizes and keeps track of all the current produce as well as their expiration dates in order to help them make decisions on when to purchase more produce.
+
+## Running/Developing
+### Setting up the repository
+Clone the repo and install the dependencies in the frontend and the backend
+```
+git clone https://github.com/TritonSE/Supply-Chain.git
+cd Suppy-Chain/backend
+npm install
+cd ../client
+npm install
+```
+
+### Running the project
+```
+cd backend npm start
+cd ../client npm start
```

Comment:
Oh hmm, well I meant them as separate commands

Reply:
That makes sense. I'm just worried that someone who's not familiar with the command line will open one terminal window and copy paste both of those lines in. Maybe something like this could work?

> To run the backend:
> ```
> cd backend
> npm start
> ```
> To run the frontend:
> ```
> cd client
> npm start
> ```

Reply:
oh good point thank you!